pydicom/tests/test_uid.py

```diff
@@ -61,7 +61,7 @@ def test_equality(self):
         assert not self.uid == 'Explicit VR Little Endian'
         # Issue 96
         assert not self.uid == 3
-        assert not self.uid is None
+        assert self.uid is not None
```

Comment:
This is no longer testing equality

Reply:
Right - this was a PEP8 warning, I overlooked that this changes the semantics.

Reply:
Fixed.